hazelcast/src/main/java/com/hazelcast/client/console/SqlConsole.java

```diff
@@ -562,6 +565,7 @@ private void stateCheck(String line, int cursor) {
         static final Integer TIMESTAMP_FORMAT_LENGTH = 19;
         static final Integer TIMESTAMP_WITH_TIME_ZONE_FORMAT_LENGTH = 25;
         static final Integer VARCHAR_FORMAT_LENGTH = 20; // it has normally unlimited precision
+        static final Integer JSON_FORMAT_LENGTH = 20;
```

Comment:
IMO this should be more, 20-char JSON is tiny. We use 25 for many other types, maybe 40 will be ok.